scm/src/GFS_typedefs.F90

```diff
@@ -2901,8 +2989,16 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     Model%cny              = cny
     Model%lonr             = gnx         ! number longitudinal points
     Model%latr             = gny         ! number of latitudinal points from pole to pole
-    allocate(Model%blksz(1:size(blksz)))
+    ! DH*
+    write(0,*) "DH DEBUG GFS_typedefs.F90: size(blksz)=", size(blksz)
```

Comment:
@climbfuji Need to leave these debug statements in?

Reply:
No. Will remove them, thanks for catching this!

Reply:
Ok, I removed those debug prints.